tools/repoman/repoman.py

```
@@ -208,9 +208,26 @@ def _sync_internal_repo(is_ci: bool) -> list[str]:
     if not _is_host_reachable(GITLAB_HOST):
         # Offline fallback: collect previously downloaded packman manifests
         manifests = []
-        for pattern in PACKMAN_GLOBS:
-            for match in glob.glob(os.path.join(REPO_ROOT, pattern)):
-                manifests.append(match)
+        cache_file = os.path.join(REPO_ROOT, ".git-blob-hash-cache.json")
+        if os.path.exists(cache_file):
+            try:
+                with open(cache_file, encoding="utf-8") as f:
+                    cache_data = json.load(f)
+                for url in cache_data.keys():
```

Ruff (0.15.5)

[error] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)